metadata.json

```diff
@@ -31,13 +31,22 @@
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
         "6",
```

Comment:
we should purge debian 6 and 7 because they are EOL.

Reply:
Yes.  I'd like to put out one release with them still in and purge them immediately after.